src/DataStructures/Tensor/Expressions/Evaluate.hpp

```diff
@@ -260,6 +237,68 @@ void evaluate(
     }
   }
 }
+}  // namespace detail
+
+/*!
+ * \ingroup TensorExpressionsGroup
+ * \brief Evaluate a RHS tensor expression to a tensor with the LHS index order
+ * set in the template parameters
+ *
+ * \details Uses the right hand side (RHS) TensorExpression's index ordering
+ * (`RhsTE::args_list`) and the desired left hand side (LHS) tensor's index
+ * ordering (`LhsTensorIndices`) to fill the provided LHS Tensor with that LHS
+ * index ordering. This can carry out the evaluation of a RHS tensor expression
+ * to a LHS tensor with the same index ordering, such as \f$L_{ab} = R_{ab}\f$,
+ * or different ordering, such as \f$L_{ba} = R_{ab}\f$.
+ *
+ * The symmetry of the provided LHS Tensor need not match the symmetry
+ * determined from evaluating the RHS TensorExpression according to its order of
+ * operations. This allows one to specify LHS symmetries (via `lhs_tensor`) that
+ * may not be preserved by the RHS expression's order of operations, which
+ * depends on how the expression is written and implemented.
+ *
+ * ### Example usage
+ * Given two rank 2 Tensors `R` and `S` with index order (a, b), add them
+ * together and fill the provided resultant LHS Tensor `L` with index order
+ * (b, a):
+ * \code{.cpp}
+ * TensorExpressions::evaluate<ti_b, ti_a>(
```

Comment:
could you point to a snippet in a test? That way at least if any part of the code implementation changes, so does the example :)

Reply:
Addressed in new fixup https://github.com/sxs-collaboration/spectre/pull/3842/commits/f4a8355967b0d42817858a11d8927ac71025017c